File: a273(walk_down_stairs)/C++/walk_down_stairs.cpp
#include <iostream>

int main(void) {
  int now, jump;
  while (std::cin >> now >> jump) {
    if (!now && !jump) std::cout << "Ok!\n";
    else if (!jump || now % jump) std::cout << "Impossib1e!\n";
    else std::cout << "Ok!\n";
  }
  return 0;
}

File: a273(walk_down_stairs)/Python/walk_down_stairs.py
from sys import stdin, stdout
while True:
    line = stdin.readline()
    if not line: break
    now, jump = map(int, line.split())
    if jump == 0 and now == 0:
        stdout.write('Ok!\n')
    elif jump == 0 or now % jump:
        stdout.write('Impossib1e!\n')
    else:
        stdout.write('Ok!\n')
